frontend/platform_specific/http/src/utils/openBrowser.ts

```
@@ -0,0 +1,4 @@
+export const openLink = (url: string) => {
+  // this is just a placeholder
```

if it's just a placeholder can we throw an exception to ensure it's not used?